lib/metanorma/cli/generator.rb

```diff
@@ -71,14 +69,18 @@ def base_templates
       end
 
       def type_specific_template
-        type_template_path = custom_template || find_template(type)
+        type_template_path = custom_template
         doctype_templates  = dir_files(type_template_path, doctype)
         build_template_hash(doctype_templates, type_template_path, doctype)
       end
 
       def custom_template
-        if options[:template]
-          Cli::GitTemplate.download(type, repo: options[:template])
+        template = options[:template]
+
+        if template && template !~ URI::regexp
+          Pathname.new(template).join(type)
+        else
+          Cli::GitTemplate.find_or_download_by(type)
```

Comment:
I can see a potential issue here, this would break for custom git template, `find_or_download_by` interface is there only to find/download the standard templates, but for the custom git template we don't want to re-use existing one but to download the latest one, and that's why we had a separate interface call `download`, which downloads a template even if that existed before.

Reply:
And the problem is the spec is currently failing. I'll leave this to you and @CAMOBAP795 to figure out then. Thanks :wink: